omero/sysadmins/server-upgrade.txt

```diff
@@ -316,6 +322,21 @@ database which can both save disk space and speed up access times.
 
     $ psql -h localhost -U **db_user** **omero_database** -c 'VACUUM FULL VERBOSE ANALYZE;'
 
+Reset ROI shape color (optional)
+""""""""""""""""""""""""""""""""
+
+Regions of interest generated using OMERO.insight did not follow the
+OME model specification. The color of shapes is now handled according
+to the data model, using RGBA rather than ARGB format.
+To upgrade the color settings of shapes created using OMERO.insight or a client
+saving them in the ARGB format, run the following script:
```

Comment:
also edit the `TRUE` in the script if you want to apply to only a subset of ROIs